### ISSUE TITLE
Error in 8muses.com

### ISSUE THREAD

**Issue by gromenauer** [yuru-yuri/manga-dl#56]
Tried:
~~~
manga-py --cbz --zero-fill --rename-pages --destination ~/Manga/ https://www.8muses.com/comics/album/BroadSword-Comics/Tarot-Witch-of-the-Black-Rose/
~~~
Error:
~~~
  File "~/.local/share/virtualenvs/0_bin-S-IZrhuG/lib/python3.5/site-packages/manga_py/providers/eight_muses_com.py", line 12, in get_archive_name
    return self.normal_arc_name(self.get_chapter_index().split('-'))
  File "~/.local/share/virtualenvs/0_bin-S-IZrhuG/lib/python3.5/site-packages/manga_py/providers/eight_muses_com.py", line 16, in get_chapter_index
    idx = re.search(self.chapter).group(1)
~~~

**Comment by 1271** [comment]
At work now. It's a pretty complicated site. :-(

**Comment by 1271** [comment]
Fixed on version 1.2.1.4

Thanks for issue!

**Comment by gromenauer** [comment]
Thanks for your hard work!
